src/test/java/seedu/duke/entries/EntryTest.java

```diff
@@ -0,0 +1,69 @@
+package seedu.duke.entries;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import seedu.duke.constants.EntryConstants;
+
+@DisplayName("Test Entry class")
+public class EntryTest {
+    private static final double AMOUNT = 2.3;
+    private static final String DESCRIPTION = "Mango pudding";
+    private static final Category CATEGORY = Category.FOOD;
+    private Entry entry;
+
+    @BeforeEach
+    void init() {
+        entry = new Entry(DESCRIPTION, AMOUNT, CATEGORY);
+    }
+
+    @Test
+    void testGetters() {
+        assertEquals(entry.getAmount(), AMOUNT);
+        assertEquals(entry.getCategory(), CATEGORY);
+        assertEquals(entry.getDescription(), DESCRIPTION);
+    }
+
+    @Test
+    void testGetCategoryString() {
+        entry.setCategory(Category.CLOTHING);
+        assertEquals(entry.getCategoryString(), EntryConstants.CLOTHING);
+        entry.setCategory(Category.ENTERTAINMENT);
+        assertEquals(entry.getCategoryString(), EntryConstants.ENTERTAINMENT);
+        entry.setCategory(Category.FOOD);
+        assertEquals(entry.getCategoryString(), EntryConstants.FOOD);
+        entry.setCategory(Category.MEDICAL);
+        assertEquals(entry.getCategoryString(), EntryConstants.MEDICAL);
+        entry.setCategory(Category.OTHERS);
+        assertEquals(entry.getCategoryString(), EntryConstants.OTHERS);
+        entry.setCategory(Category.PERSONAL);
+        assertEquals(entry.getCategoryString(), EntryConstants.PERSONAL);
+        entry.setCategory(Category.TRANSPORTATION);
+        assertEquals(entry.getCategoryString(), EntryConstants.TRANSPORTATION);
+        entry.setCategory(Category.UTILITIES);
+        assertEquals(entry.getCategoryString(), EntryConstants.UTILITIES);
```

Comment:
Perhaps we could consider an `enum` to store these predefined expense categories such that testing is made easier? 

With that, we can create a method to loop through each category within its `enum` and then pass it into `assertEquals()` for testing. This also makes future testing more convenient if more categories were to be added.

Reply:
Great point. This is highlighted in #58 as part of future code improvements. Will mark this comment as resolved.